iFixit/react-commerce
# Add product links to featured product list section

This PR closes #110 

## Changelog

- Adds product links to FeaturedProductList section products
- Refactors product cards to match new design
- Renames some `collection` folder names to now use `productList` term instead

## QA

Until we setup the staging servers we need to clone the repo and test it locally.

1. Clone this repo
2. Make sure you are running on node v14 (if you are using nvm run `nvm use` in the project working directory)
3. Install dependencies `npm install`
4. Rename `backend/.env.example` to `backend/.env` (this will enable db seed)
5. Run dev server: `npm run dev`
6. Login to strapi at `localhost:1337/admin` using the credentials specified below
6. Navigate to the `parts` product list and add a `FeaturedProductList` section (e.g. Macbook parts)
7. Visit the parts collection at `localhost:3000/collections/parts`
8. Verify that products within the featured product list correctly link to product pages:
    <img width="883" alt="Screenshot 2021-11-18 at 11 08 48" src="https://user-images.githubusercontent.com/4640135/142395255-5e4b322a-220d-4866-abd6-c4a8d6e1b98c.png">

### Access to local strapi instance

Email: `strapi@ifixit.com`
Password: `Password1`
